.github/workflows/libjs-test262.yml

```diff
@@ -114,6 +115,32 @@ jobs:
             --per-file-output ../libjs-website/test262/data/per-file-master.json \
             --per-file-bytecode-output ../libjs-website/test262/data/per-file-bytecode-master.json
 
+      - name: Run test-wasm
+        working-directory: libjs-test262/Build
+        run: |
+          _deps/lagom-build/test-wasm --detailed-json > ../../libjs-website/wasm/data/per-file-master.json
+          jq -nc -f <<-EOF --slurpfile previous ../../libjs-website/wasm/data/results.json --slurpfile details ../../libjs-website/wasm/data/per-file-master.json
```

Comment:
Oh I just kinda assumed it'd be there, I'll add it to the apt install above (does the runner have some package cache? redownloading the same stuff is pretty silly for a server not in a data center...)

Reply:
If the package is already installed it would just say "already installed" and skip it, it's just a plain old ubuntu box (so the apt step is kinda redundant after the first time, aside from upgrades)